simplipy/websocket.py

```diff
@@ -190,6 +193,23 @@ def __post_init__(self, event_cid: int) -> None:
                 )
                 object.__setattr__(self, "sensor_type", None)
 
+        if self._vid is not None and self._video is not None:
+            object.__setattr__(
+                self,
+                "media_urls",
+                {
+                    "imageUrl": self._video[self._vid]["_links"]["snapshot/jpg"][
+                        "href"
+                    ],
+                    "clipUrl": self._video[self._vid]["_links"]["download/mp4"]["href"],
+                    "hlsUrl": self._video[self._vid]["_links"]["playback/hls"]["href"],
+                },
+            )
+            object.__setattr__(self, "_vid", None)
```

Comment:
Why do we care about setting these two to `None`?

Reply:
If not set to None, they make it into the WebsocketEvent data, and seemed yuky to be sending to clients.